### DISCUSSION TITLE
Location Hierarchy proposal

### DISCUSSION THREAD

**Discussion by WolfgangFahl** [somnathrakshit/geograpy3#35] [Ideas]
For our current OpenResearch use case we came up with:

![grafik](https://user-images.githubusercontent.com/1336221/120319039-d8299a00-c2e0-11eb-9a44-7fd7326eeeb8.png)
see http://wiki.bitplan.com/index.php/Location_Hierarchy

to fix our issue https://github.com/SmartDataAnalytics/OpenResearch/issues/220 which is a fix for https://github.com/SmartDataAnalytics/OpenResearch/issues/209. We are tentatively thinking to move the code from https://github.com/SmartDataAnalytics/OpenResearch/blob/master/migration/openresearch/location.py here. 
**What do you think?**

**Comment by WolfgangFahl** [comment]
![grafik](https://user-images.githubusercontent.com/1336221/120324092-8257f080-c2e6-11eb-91ab-bad0198ca693.png)

can be created by the test case test_locator/test_population as a side effect

**Comment by tholzheim** [comment]
See https://confident.dbis.rwth-aachen.de/orth/index.php?title=List_of_Locations for an example of the location hierarchy
